src/assets/CloudinaryFile.ts

```diff
@@ -138,6 +140,14 @@ class CloudinaryFile {
     return this;
   }
 
+  /**
+   * Sets the __cld_token__ value for the authToken signature
+   * @param token
+   */
+  setExplicitAuthToken(token: string) {
```

Comment:
`setAuthToken` sounds like a better name, the word `Explicit` only introduces confusion.

Reply:
The problem here is that the word `authToken` already signifies the whole authToken config, see this: 

https://github.com/cloudinary/js-url-gen/pull/519/files/2cabdf9204ce87ad7c98d1dbafd488c09827017f#diff-bcfa6f78219d31a71565b9004ec7f94c114978d695f62b52c18dd6e759d6d9c7R61

It's a bit strange that setAuthToken here is a string, while authToken is a configuration object.
Perhaps the config name should change to `authTokenConfig`, and this one should change to `setAuthToken`?

What do you think?